client/settings/settings-manager/index.js

```diff
@@ -2,16 +2,20 @@
 /**
  * External dependencies
  */
+import { Button, ExternalLink } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import React, { useState } from 'react';
-import { ExternalLink } from '@wordpress/components';
+import React from 'react';
+
 /**
  * Internal dependencies
  */
+import './style.scss';
+import { LoadableBlock } from '../../components/loadable';
+import { useSettings } from '../../data';
 import Banner from '../../banner';
 import PaymentMethods from '../../payment-methods';
-import DigitalWallets from '../digital-wallets';
 import SettingsSection from '../settings-section';
+import DigitalWallets from '../digital-wallets';
```

Comment:
```suggestion
import DigitalWallets from 'settings/digital-wallets';
```

Reply:
More absolute paths suggestions... Please feel free to reject them if you do not like them

Reply:
Thanks! I wasn't super familiar with the consequences of using absolute paths and I was worried there might be some issues with IDE support and resolving where a path points to so I used relative ones for unambiguous resolution.

I read up a bit on the subject and there is less magic than I thought, plus there are some nice benefits like cleaner paths and not having to change paths when moving files, so I think using absolute paths is a great idea!